pavelib/utils/envs.py

```diff
@@ -203,7 +203,8 @@ class Env(object):
     # Directories used for common/lib/ tests
     LIB_TEST_DIRS = []
     for item in (REPO_ROOT / "common/lib").listdir():
-        if (REPO_ROOT / 'common/lib' / item).isdir():
+        dir_name = (REPO_ROOT / 'common/lib' / item)
+        if dir_name.isdir() and not dir_name.endswith(('__pycache__', '.cache')):
```

Comment:
Nit: Is there any likelihood that this list of ignored dirs might change? If so maybe move it to a more obvious / configurable place?

Reply:
Makes sense - `__pycache__` is already in two places in the paver code. I'll move to a common location.